Documents/UseCasesForWebsite.md

```diff
@@ -0,0 +1,39 @@
+Actors: Visitor, Artist, Admin, System 
+
+**Visitor** - A person which is visiting the website. 
+A visitor would use this website for getting information or donate as amputee a prosthetic or donate as a company. 
+Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- **can request for a prosthetic**. Then the visitor needs to fill in a form, which can be reached by a link in the menu,  where the name, email-address and sort of prosthetic is filled in. 
+  When the visitor wants to pain the prosthetic then also the visitor can choose for a artist which the visitor likes or add some sketches of arts the visitor wants on the prosthetic.
+- **to donate a prosthetic, money or attention/advertisement** he/she fill in a form which can be found at the donate link in the menu of the site. The visitor needs to fill in a name, email address
+    maybe a telephone number and the visitor can choose by a select list what the visitor wants to donate. 
+- **goto the public gallery page** which can also be found with a link on the menu of the site. The images will be sorted with the latest one first. 
+- **View info page** by following the link on the menu of the website. On the info page the visitor can read information about the mission of this project and how it�s started, 
+   success stories and comments of the artist involved. 
+- **Like a picture**. When a visitor likes a picture, he/she can click on a like button. 
+- **View artwork**. When the visitor goes to the public gallery by clicking on the gallery link. When the visitor then clicks on a image shown in the gallery a modal screen is shown zoomed 
```

Comment:
How to get to a gallery, is **View gallery** concern. How to view artwork?- you just need to be in a gallery.
How to get to gallery?- you click the link- but it's not important for this use case.